src/xdg.c

```diff
@@ -526,6 +524,7 @@ xdg_surface_new(struct wl_listener *listener, void *data)
 	view->type = LAB_XDG_SHELL_VIEW;
 	view->impl = &xdg_toplevel_view_impl;
 	xdg_toplevel_view->xdg_surface = xdg_surface;
+	view->ssd_preference = LAB_SSD_PREF_CLIENT;
```

Comment:
Should we explain why we set `ssd_preference` to client here? I get it following IRC discussion, but it's pretty non-intuitive.

Reply:
Added some more comments